tests/summarizer/test_image_description_retreiver.py

```diff
@@ -95,16 +141,22 @@ def test_web_entity_detection(mocked_post):
         ["https://tinyurl.com/y7how2rj"])[0]
     assert "sundar pichai" in reponse_for_image_url_1["label"]
     assert "sundar pichai Alphabet" in reponse_for_image_url_1["entities"]
+    assert reponse_for_image_url_1["has_caption"]
```

Comment:
Also test that has_caption is filled correctly according to word_count. Appologies if this is already done.

Reply:
Yes! do take a look at the API mock defined above - there is a sentence with the number of words > word_count_to_qualify_as_caption